pkg/ingress/config/ingress_config.go

```diff
@@ -311,6 +311,9 @@ func (m *IngressConfig) List(typ config.GroupVersionKind, namespace string) []co
 		}
 		configs = append(configs, configsFromGateway...)
 	}
+	if configsFromMcpBridges := m.listFromMcpBridge(typ); configsFromMcpBridges != nil {
```

Comment:
The `listFromIngressControllers` will also call `convertDestinationRule` and `convertServiceEntry`, where the dr and se from mcpbridge will be taken out again. Wouldn't directly listing all and appending them here cause duplication?